atlas/atlas.star

```diff
@@ -0,0 +1,175 @@
+"""
+A URL is an absolute location within the atlas.
+"""
+scalar URL
+
+"""
+Names are dotted paths of GraphQL identifiers, like `spotify.User`
+and `String`.
+"""
+scalar Name
+
+type Query {
+  at(id: URL): Point
+  edge(name: Name, from: Name, to: Name): Edge
+  type(id: URL): Type
+  interface(id: URL): Interface
+  edge(id: URL): Edge    
+  points: [Point]
+}
+
+## This would be cool:
+# interface Point @key(fields: "id", via: "Query.at(id)") {
```

Comment:
What does the `via` do here? Is that meant to represent its canonical fetch point in the graph? How does this differ from the current `_entities` entry point design?